pkg/controller/ingress/ingress_controller_test.go

```diff
@@ -54,6 +55,63 @@ var (
 	}
 )
 
+func TestRouteMigration(t *testing.T) {
+	logf.SetLogger(logf.ZapLogger(true))
+
+	tests := []struct {
+		name        string
+		exRoute     *routev1.Route
+		wantName    string
+		removedName string
+	}{
```

Comment:
```suggestion
	}{{
```

Spares you one level of indentation.